### ISSUE TITLE
Highlight top spots and black holes

### ISSUE THREAD

**Issue by tillwenke** [hitchmap/hitchmap#69]
see https://github.com/Hitchwiki/hitchmap-data/blob/main/visualization/data_analysis.ipynb

**Comment by bopjesvla** [comment]
This is gonna be tricky from a UX perspective. I hope the route planner will make this unnecessary as it kinda does this already

**Comment by tillwenke** [comment]
yeah. thought of it more as a gimmick. prolly a good first place for this is the dashboard.
